### DISCUSSION TITLE
봇 상세페이지에 편의성을 더한 접근방식

### DISCUSSION THREAD

**Discussion by getsolaris** [koreanbots/koreanbots#319] [제안] [answered]
안녕하세요.

한국 디스코드봇 리스트에 많은 관심을 가지고 있는 이용자입니다.

다름이 아니라, `한국 디스코드봇 리스트` 사이트에서 봇 상세페이지에 들어가기 위해서는 `보기` 버튼을 눌러서 들어가는 방법이 있습니다.

하지만, `보기` 버튼을 대신하여 `content` class 를 클릭 시 상세페이지에 접근하는 방법으로 변경한다면 사용자의 편의가 더해질 것 같습니다.

후 `보기` 버튼을 제거하여, `초대하기` 버튼의 사이즈가 커졌으면 좋겠습니다.

저의 접근 방식이 틀렸다면 알려주시면 감사하겠습니다. :)

---

1. `content` class 클릭 시 상세페이지 이동
2. `초대하기` 버튼을 한 눈에 볼 수 있도록, `보기` 버튼 제거

**Comment by wonderlandpark** [comment, marked as answer]
안녕하세요 :)

먼저, 한국 디스코드봇 리스트에 관심가져주셔서 감사합니다.

현재 한국 디스코드봇 리스트는 재작성 중입니다. 따라서 아래와 같이 디자인을 변경하였습니다.

https://user-images.githubusercontent.com/31924512/114261268-9ed95a00-9a14-11eb-9ac0-67b879bc548e.mov

제안해주신 `content` class를 클릭하여 봇 페이지로 이동하는 방식은 구현되어있는 것 같습니다.

보기 버튼을 제거하고 초대하기 버튼만 남기는 방안은 추후 A/B 테스트를 진행하는 등, 긍정적으로 검토해보도록 하겠습니다. 의견 남겨주셔서 감사합니다.

**Reply by getsolaris** [reply]
답변 주셔서 감사합니다 !

점점 더 이뻐지고 있는 것 같아 보기 좋은 것 같습니다.
